index.html

```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
-          <div>
-            <ul id="profs">
-              <li id="cannopy">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              A daily selection of privately personalized: no accounts or
-              sign-ups required
-            </p>
-          </div>
-          <div>
-            <ul class="button">
-              <li><button type="button">html</button></li>
-              <li><button type="button">css</button></li>
-              <li><button type="button">javascript</button></li>
-            </ul>
-          </div>
-          <div class="hidden3">
-            <button type="button">See Project</button>
-          </div>
-        </div>
-        <div class="tonic">
-          <img src="images/Snapshoot-Portfolio-2.svg" alt="snap2" />
-          <h2 id="stories-header">Multi-post Stories</h2>
-          <ul id="profs1">
-            <li id="cannopy1">CANOPY</li>
-            <li>Back End Dev</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized: no accounts or sign-ups
-            required
-          </p>
-          <ul class="button">
-            <li><button type="button">html</button></li>
-            <li><button type="button">css</button></li>
-            <li><button type="button">javascript</button></li>
-          </ul>
-          <div class="hidden1">
-            <button type="button">See Project</button>
-          </div>
-        </div>
-        <div class="tonic">
-          <div><img src="images/Snapshoot-Portfolio-3.svg" alt="snap3" /></div>
-          <div>
-            <h2 id="stories-header1">Tonic</h2>
-          </div>
-          <div>
-            <ul id="profs2">
-              <li id="cannopy2">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              A daily selection of privately personalized: no accounts or
-              sign-ups required
-            </p>
-          </div>
-          <div>
-            <ul class="button">
-              <li><button type="button">html</button></li>
-              <li><button type="button">css</button></li>
-              <li><button type="button">javascript</button></li>
-            </ul>
-          </div>
-          <div class="hidden2"><button type="button">See Project</button></div>
+      <section id="works">
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup starts from Here -->
+
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+            <div class="desktopIntro">
+              <h2 class="deskHeading"></h2>
+              <!-- <ul class="deskButtons">
+                <li class="deskList1"></li>
+                <li class="deskList2"></li>
+                <li class="deskList3"></li>
+              </ul> -->
```

Comment:
- [ ] Kindly remove this commented code to improve readability and make your code more professional 💯